.github/workflows/ci.yml

```
@@ -25,33 +24,34 @@ jobs:
       - name: Lint
         run: pnpm lint
       - name: Run Tests
-        run: pnpm test:ember
+        run: pnpm test
 
   floating:
     name: "Floating Dependencies"
     runs-on: ubuntu-latest
-    timeout-minutes: 10
 
     steps:
       - uses: actions/checkout@v4
       - uses: wyvox/action-setup-pnpm@v3
         with:
           node-version: 18
-          args: '--no-lockfile'
       - name: Run Tests
-        run: pnpm test:ember
+        run: pnpm test
 
   try-scenarios:
     name: ${{ matrix.try-scenario }}
     runs-on: ubuntu-latest
-    needs: "test"
-    timeout-minutes: 10
+    needs: 'test'
```

Can we please reset to double quotes? This is how Prettier is configured for YAML files. Even though we don't check them in CI.

```suggestion
    needs: "test"
```